view/theme/frio/js/modal.js

```diff
@@ -293,7 +293,7 @@ function editpost(url) {
 
 			// To make dropzone fileupload work on editing a comment, we need to
 			// attach a new dropzone to modal
-			dzFactory.setupDropzone('#jot-text-wrap', 'profile-jot-text'); 
+			dzFactory.setupDropzone('#jot-text-wrap', 'profile-jot-text');
```

Comment:
I didn't intend to touch this line, but apparently I manually edited this file at some point and my editor (VSCode) insisted on removing this space. I'm sorry. I can un-remove the space...